src/components/Header/index.js

```diff
@@ -1,20 +1,77 @@
-import React from "react";
+import React, { useState } from "react";
 import style from "./styles";
 import { Helmet } from "react-helmet";
+import logo from "../../images/logo.png";
+import logo_desk from "../../images/logo_desk.png";
+import { ThreeLineHorizontal } from 'akar-icons';
+import { Link } from 'react-scroll';
+import { useSpring, animated, useSpringRef, useChain } from 'react-spring'
 
-const Header = ({ className }) => (
-  <>
-    <Helmet>
-      <meta charSet="utf-8" />
-      <title>Impulsiona Seguidores</title>
-      <link rel="preconnect" href="https://fonts.gstatic.com"></link>
-      <link
-        href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
-        rel="stylesheet"
-      ></link>
-    </Helmet>
-
-  </>
-);
+
+
+const Header = ({ className }) => {
+    const [check, setCheck] = useState(false);
```

Comment:
Try to conceive clearer variable names. Generic names, such as "check" or "flag" are generally bad, because we can't understand their purpose by reading the code. If in this case the variable is to control whether we should show a menu or not, "showMenu" could be a nice example of a more understandable name.

Reply:
Fixed in 693fa86